scripts/qa_crowbarsetup.sh

```diff
@@ -1711,6 +1713,10 @@ function custom_configuration()
             proposal_set_value nova default "['attributes']['nova']['use_migration']" "true"
             [[ "$libvirt_type" = xen ]] && sed -i -e "s/nova-multi-compute-$libvirt_type/nova-multi-compute-xxx/g; s/nova-multi-compute-kvm/nova-multi-compute-$libvirt_type/g; s/nova-multi-compute-xxx/nova-multi-compute-kvm/g" $pfile
 
+            if [[ $all_with_ssl = 1 || $nova_with_ssl = 1 ]] ; then
+                enable_ssl_for_nova
+            fi
```

Comment:
This sounds unrelated to this PR. Error during a rebase? Or github displaying weird bits?